extra/release_tool.py

```diff
@@ -2460,17 +2460,30 @@ def log_and_run_cmd(cmd, workdir):
         fd.write(changelog_out.decode())
 
 
-def do_generate_release_notes(state):
-    """Generate Release Notes for the released tags"""
```

Comment:
Q:
Why remove the comment all together?

Reply:
Brainfart. I have no idea why I removed it, I will put it back.